translations/mod-fqm-manager/en.json

```diff
@@ -1,4 +1,23 @@
 {
+  "entityType..code": "Code",
```

Comment:
These should be removed (I'm guessing these were for service points but you hadn't named the entity type yet)